README.md

```diff
@@ -1,3 +1,7 @@
-# abap-trace-tender
+# abap-trace-tender :construction: WIP
 
 Just an idea for now. More hopefully in the future...
+
+# architecture
+
+![architecture](./out/architecture/architecture/architecture.png)
```

Comment:
Is there a reason for two times architecture as subfolder?

Reply:
It's copied form here: https://github.com/abap-observability-tools/abap-log-exporter/tree/main/out/architecture/architecture 😄 
I didn't really try to change that. I assume that depends also on how the output of plantuml works.